### PR TITLE
Cleans code style of '/src/containers'. #1018

### PR DESCRIPTION
**Pull request by knod**

### CODE REVIEW

File: src/containers/VisitPage.js [hunk 1]
```diff
@@ -271,78 +277,74 @@ class VisitPage extends Component {
...
+          shouldRequestConfirmation = { this.shouldConfirmLeave }
+          isBlocking                = { this.state.isBlocking } />
 
         {/* = LINKS? = */}
         {/* We should probably remove this. If we want to
          * do this we might do this a different way at this
          * point. Perhaps a user's page should be a route
          * in VisitPage? */}
-        { this.state.redirect ? (
+        { (this.state.redirect) ? (
```
Comment (turnerhayes) [inline src/containers/VisitPage.js]: Why add the extra parens?

File: src/containers/VisitPage.js [hunk 2]
```diff
@@ -355,20 +357,18 @@ class VisitPage extends Component {
...
-          distrustConfirmed === false ? (
-            <PredictionsWarning
-              distrustConfirmed       = { distrustConfirmed }
-              toggleDistrustConfirmed = { this.props.funcs.toggleDistrustConfirmed }
-              translations={{ ...translations.warningModal }} />
-          ) : (
-            null
-          )
-        }
+        { (distrustConfirmed === false) ? (
```
Comment (turnerhayes) [inline src/containers/VisitPage.js]: (same question as above)

File: src/containers/VisitPage.js (hunk 1 above)
Comment (knod) [reply src/containers/VisitPage.js]: Well, in more complex conditionals I find it easier to read if there are parens around the conditional. If I'm going to do it elsewhere, I thought I'd try being consistent and do it here too. Thoughts?

File: src/containers/VisitPage.js (hunk 2 above)
Comment (knod) [reply src/containers/VisitPage.js]: Same answer. This isn't a super complex one, though it's a bit more complex and the parens help even here. I also think it makes it look even more like a regular if statement, which is what I find easiest to read.

File: src/containers/VisitPage.js (hunk 1 above)
Comment (turnerhayes) [reply src/containers/VisitPage.js]: I see. I kind of feel like it clutters things up to have extra parens. I think for conditionals that don't involve operations or multiple lines (for the conditional part), I would prefer leaving off the parens.

File: src/containers/VisitPage.js (hunk 2 above)
Comment (knod) [reply src/containers/VisitPage.js]: @turnerhayes : Would we remove the parens here, or do they stay because this has an operation?

**Review by dylanesque — APPROVED** [review]

File: src/containers/VisitPage.js (hunk 2 above)
Comment (turnerhayes) [reply src/containers/VisitPage.js]: Debatable--I would say equality is a simple enough operation, but that does get into the question of "what's simple enough" and that's probably not something we want to wade into.

Would this be okay to replace with `!distrustConfirmed`? Or can it have falsy-but-not-`false` values?

Comment (knod) [reply src/containers/VisitPage.js]: I can switch it up

Comment (knod) [reply src/containers/VisitPage.js]: I don't think it has falsy/truthy values...

**Review by turnerhayes — APPROVED** [review]

**Pull request merged**